Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -765,6 +770,6 @@ extension BTPayPalClient: BTAppContextSwitchClient {
     /// :nodoc:
     @_documentation(visibility: private)
     @objc public static func canHandleReturnURL(_ url: URL) -> Bool {
-        BTPayPalReturnURL.isValid(url)
+        BTPayPalReturnURL.isValid(url, fallbackURLScheme: payPalClient?.fallbackURLScheme)
```

Comment:
small nit: I believe we should just be able to access the property directly here vs using the optional client:

```suggestion
        BTPayPalReturnURL.isValid(url, fallbackURLScheme: fallbackURLScheme)
```

Reply:
As `canHandleReturnURL` is a static method and `fallbackURLScheme` is an instance property, I think we need to access it through the static `payPalClient` reference. Otherwise: "Instance member 'fallbackURLScheme' cannot be used on type 'BTPayPalClient'"